src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2259,6 +2260,9 @@ export class StudyViewPageStore {
     });
 
     @computed get oncokbCancerGeneFilterEnabled() {
+        if (AppConfig.serverConfig.species === "mouse") {
```

Comment:
@jjgao @inodb is it correct that a whole portal instance could be species specific and should it be treated as a configuration property like this?

Reply:
@alisman the species is in the backend. Currently the database schema would allow for different species in one instance:

```
mysql> select * from reference_genome;
+---------------------+---------+------+------------+-------------+---------------------------------------------------------+---------------------+
| REFERENCE_GENOME_ID | SPECIES | NAME | BUILD_NAME | GENOME_SIZE | URL                                                     | RELEASE_DATE        |
+---------------------+---------+------+------------+-------------+---------------------------------------------------------+---------------------+
|                   1 | human   | hg19 | GRCh37     |  2897310462 | http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips  | 2009-02-01 00:00:00 |
|                   2 | human   | hg38 | GRCh38     |  3049315783 | http://hgdownload.cse.ucsc.edu/goldenPath/hg38/bigZips  | 2013-12-01 00:00:00 |
|                   3 | mouse   | mm10 | GRCm38     |  2652783500 | http://hgdownload.cse.ucsc.edu//goldenPath/mm10/bigZips | 2012-01-01 00:00:00 |
+---------------------+---------+------+------------+-------------+---------------------------------------------------------+---------------------+
```
I don't think anyone is using it like that at the moment, so a single portal instance per species (or genome build for that matter) probably makes sense, but you should be able to retrieve for a particular study whether it is mouse or human using the API given this db schema. So maybe better to use an endpoint instead of using `serverConfig`

Maybe @n1zea144 or @jjgao can elaborate

Reply:
@JiaoJiao123 @n1zea144 please make call on whether this important

Reply:
I don't think our code frontend/backend code can support multiple species, e.g. how to deal with the same gene symbol in different species?

@oplantalech can we not add the `species` property and use oncokb-related properties to turn off oncokb for mouse instance? Ideally we should remove the mouse reference genome from the human seed database as well.